packages/palette/package.json

```diff
@@ -26,7 +26,10 @@
     "type-check": "tsc --emitDeclarationOnly --pretty",
     "type-declarations": "tsc --emitDeclarationOnly",
     "watch": "concurrently --raw --kill-others 'yarn compile -w' 'yarn type-declarations -w'",
-    "watch:emission": "yarn clean:emission && concurrently --raw --kill-others 'yarn compile:emission -w' 'yarn type-declarations -w --outDir ../../../emission/node_modules/@artsy/palette/dist'"
+    "watch:emission": "yarn clean:emission && concurrently --raw --kill-others 'yarn compile:emission -w' 'yarn type-declarations -w --outDir ../../../emission/node_modules/@artsy/palette/dist'",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
```

Comment:
Is this accessible via netlify too?

Reply:
It's not enabled right now. Chromatic gives a storybook preview of every deploy though, so we could probably lean on that.